praw/models/reddit/mixins/__init__.py

```diff
@@ -139,6 +139,61 @@ def remove(self, spam=False):
         data = {"id": self.thing.fullname, "spam": bool(spam)}
         self.thing._reddit.post(API_PATH["remove"], data=data)
 
+    def removal_reasons(self):
+        """Return a dict of available removal reasons.
+
+        Reason ids are required in order to use :meth:`.add_removal_reason`.
+
+        The dict contains two keys: "data" and "order"
+        - The "data" key is a dict of reasons with ID as the key,
+        and "message", "id", and "title" as keys in the sub-dict.
+        - The "order" key is a list of reason IDs in the display order
+        when selecting on Reddit.
+
+        For example:
+
+        .. code:: python
+
+           removal_reasons = submission.mod.removal_reasons()
+
+        """
+        url = API_PATH["removal_reasons_list"].format(
+            subreddit=self.thing.subreddit
+        )
+        return self.thing._reddit.get(url)["data"]
+
+    def add_removal_reason(self, reason_id, mod_note=""):
+        """Add a removal reason for a Comment or Submission.
+
+        :param reason_id: The removal reason template ID.
+        :param mod_note: A message for the other mods on Reddit redesign
```

Comment:
Can we change this description to `A message for the other moderators.` including ending period?